### ISSUE TITLE
Compatibility with Spacy 3+

### ISSUE THREAD

**Issue by sarkarnayan** [huggingface/neuralcoref#348]
Is this compatible with Spacy 3+ ?

**Comment by plandes** [comment]
I agree.  The majority of other packages that use spaCy are up to at least 3.2.  As of this writing, [3.4](https://spacy.io/usage/v3-4) is the current version.

**Comment by polm** [comment]
This package is not compatible with spaCy v3, but we just released [a new coreference model](https://explosion.ai/blog/coref) that is compatible.

**Comment by VaibhaveS** [comment]
@polm 
I keep getting this error while trying it out. Any pointers please?
`
coref = nlp.add_pipe("experimental_coref")
`
RegistryError: [E893] Could not find function 'spacy-experimental.Coref.v1' in function registry 'architectures'. If you're using a custom function, make sure the code is available. If the function is provided by a third-party package, e.g. spacy-transformers, make sure the package is installed in your environment.

I have installed spacy-experimental

**Comment by polm** [comment]
In case anyone else comes here, please note we have followed up on this question here: https://github.com/huggingface/neuralcoref/issues/348

In general, it's better to follow up with issues in the repo that applies to the code they come from.

**Comment by MeatballCat** [comment]
> This package is not compatible with spaCy v3, but we just released [a new coreference model](https://explosion.ai/blog/coref) that is compatible.

I downloaded your latest pronoun reduction model. Can it output the sentence after pronoun reduction directly?

**Comment by polm** [comment]
Please post questions about the new coref model in the spaCy repo, and not here. Your question has been asked at https://github.com/explosion/spaCy/discussions/11585#discussioncomment-3970887, for example.

**Comment by MeatballCat** [comment]
> Please post questions about the new coref model in the spaCy repo, and not here. Your question has been asked at [explosion/spaCy#11585 (reply in thread)](https://github.com/explosion/spaCy/discussions/11585#discussioncomment-3970887), for example.

thanks for your reply
